Dockerfile

```diff
@@ -42,12 +42,12 @@ FROM builder AS lint_yaml
 RUN find . -type f \( -name '*.yml' -o -name '*.yaml' \) -exec yamllint {} +
 
 ## Type Annotations Linter
-FROM builder AS lint_types
-CMD find "${ENV_VENDOR}" -type f -name '*.py' -exec mypy {} +
+#FROM builder AS lint_types
```

Comment:
Placed on the backlog to undisable and add to `make lint`, but didn't want to leave the misleading `CMD` statements here, as they aren't being executed based on the current `Makefile`